### PR TITLE
HOLD - DEVELOPER-3088 - Added in the ability for SEO content editors to modi…

### PR DESCRIPTION
**Pull request by Dantheman720**
@LightGuard - So, I got this to work on my machine locally. Does this work for you as well?

### CODE REVIEW

**Comment by Dantheman720** [comment]
@robpblake - Hey Rob,  I wanted to make sure that this works locally for you as well.

**Comment by robpblake** [comment]
@Dantheman720 How do I verify that this is working for me? Can you give me some instructions on how to test it?

cheers,

Rob

**Comment by LightGuard** [comment]
This is a good start, but you're missing the defaults for all the pages.

File: _docker/drupal/README.md [hunk 1]
```diff
@@ -106,6 +106,28 @@ section of composer.json:
...
+
+Metatag is used to automatically generate structured metadata. To manipulate generated metadata, you will need to work with the Metatag module and export the settings.
+
+1. Locally spin up an instance of drupal (i.e. bundle exec ./control.rb -u --run-the-stack)
+2. Login to drupal with the admin username and password.
+3. Navigate to Configuration>Search and metadata>Metatag 
+4. Select the 'Edit' button next to either 'Content: Article' or 'Content: Topic' and manipulate data to fit requirements. (Browse all available tokens can be helpful when working with dynamic variables)
+5. Save all changes.
+
+Now that all changes have been saved, it is necessary to export the configuration in order for subsequent builds to contain the changes.
```
Comment (LightGuard) [inline _docker/drupal/README.md]: This is not needed, though having something about having to edit the global defaults would be good.

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

**Comment by robpblake** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
retest this please

**Pull request merged**